gitlab/datadog_checks/gitlab/gitlab.py

```diff
@@ -81,6 +79,13 @@ def _create_gitlab_prometheus_instance(self, instance, init_config):
         # gitlab uses 'prometheus_endpoint' and not 'prometheus_url', so we have to rename the key
         gitlab_instance['prometheus_url'] = instance.get('prometheus_url', instance.get('prometheus_endpoint'))
 
+        url = gitlab_instance.get('gitlab_url')
+
+        host_tags = self._check_tags(url)
+        tags = gitlab_instance.get('tags', [])
+        host_tags.extend(tags)
+        self._tags = host_tags
```

Comment:
Can we simplify this section a bit? Could `self._check_tags()` take the instance and just return a list of tags?

Reply:
Sure!

Reply:
Updated!